go.sum

```diff
@@ -1069,6 +1071,8 @@ github.com/otiai10/curr v0.0.0-20190513014714-f5a3d24e5776/go.mod h1:3HNVkVOU7vZ
 github.com/otiai10/mint v1.2.3/go.mod h1:YnfyPNhBvnY8bW4SGQHCs/aAFhkgySlMZbrF5U0bOVw=
 github.com/otiai10/mint v1.2.4/go.mod h1:d+b7n/0R3tdyUYYylALXpWQ/kTN+QobSq/4SRGBkR3M=
 github.com/otiai10/mint v1.3.0/go.mod h1:F5AjcsTsWUqX+Na9fpHb52P8pcRX2CI6A3ctIT91xUo=
+github.com/p-lambert/go-re v0.0.0-20200818160800-5353e43b34bc h1:dE3PAD9vPzB/+L2LyfKtUr/beBBsxENkc7NbgHDafKc=
+github.com/p-lambert/go-re v0.0.0-20200818160800-5353e43b34bc/go.mod h1:dqOKqIR215VpOgeINQHSP50lCOlVt/Z7olxvOgO1iZU=
```

Comment:
My preference would be for this to be included in the code of `datadog-agent`. Otherwise there is a lot of churn every time we need to make any changes to the compiler.